dallinger/experiment.py

```diff
@@ -1197,9 +1205,9 @@ def monitoring_statistics(self, **kw):
         :returns: An ``OrderedDict()`` mapping panel titles to data structures
                   describing the experiment state.
         """  # noqa
-        participants = Participant.query
-        nodes = Node.query
-        infos = Info.query
+        participants = db.session.query(Participant)
```

Comment:
It shouldn't matter, but it seems strange that we're using `db.session` here and `self.session` elsewhere. Is there a reason for that?

Reply:
Oversight on my part, but relevant to the above discussion :)